utils/sl/env_modules.py

```diff
@@ -59,24 +59,25 @@ def _module_func(self, command, *arguments): # pylint: disable=no-self-use
 
         try:
             proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
+        except FileNotFoundError:
+            return None, None
```

Comment:
I'm not sure about this one, I've a feeling it's this way round for a reason.  We've certainly had one piece of code like this that needed to do the excepts in the correct order for python2/python3 compatibility.  The two blocks are checking the same thing, just for different versions so it may be that we can simply remove the OSError one, but it would need testing.

Reply:
Will remove the OSError codes in the next build for testing, will restore it if failed.

Reply:
It may be that we only test python2 or python3 and developers will have a mix, and that CI will have the required packages installed so this error would never happen.  It will need manual testing.

Reply:
The original error was due to mis-order.
Actually after removed the redundant   "except FileNotFoundError: ...", all tests passed..
We can moving forward on this one.
Thanks.